Weather Alert/Networking/Networking.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Networking.swift
+//  Weather Alert
+//
+//  Created by Jack Hopkins on 18/11/2016.
+//  Copyright © 2016 UKFast. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+open class NetworkingRequest {
+    
+    // MARK: Variables
+    
+    public var path: String
+    public var response: [String : Any]?
+    
+    var networking: Networking { return AppShared.instances.networking }
+    var dataController: DataController { return AppShared.instances.dataController }
+    
+    init() {
+        path = "weather?id=1000006"
```

Comment:
Better than just having an empty string, an error should be thrown (or aborting process) if this is not set/overridden.